projects/igniteui-angular/src/lib/grids/filtering/excel-style/grid.excel-style-filtering.component.ts

```diff
@@ -427,6 +428,7 @@ export class IgxGridExcelStyleFilteringComponent implements OnDestroy, AfterView
         selectAll.value = this.grid.resourceStrings.igx_grid_excel_select_all;
         selectAll.label = this.grid.resourceStrings.igx_grid_excel_select_all;
         selectAll.indeterminate = this.selectAllIndeterminate;
+        selectAll.isSpecial = true;
```

Comment:
We should do this for the '**blanks**' filter item as well (as it is done in the other PR)